go.mod

```diff
@@ -32,3 +32,5 @@ require (
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
 	google.golang.org/grpc v1.30.0
 )
+
+replace github.com/tendermint/tendermint => ./
```

Comment:
This is so we don't have to change all the imports?

Reply:
Yep. We could do a hard fork but then backporting will be much more complicated. We could try to stay with this approach since it works and decided what is better later. What do you think?